routes/auth.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const config = require('config');
+const auth = require('../middleware/auth');
+const { check, validationResult } = require('express-validator');
+const _ = require('lodash');
+
+const { User } = require('../sequelize');
+
+router.get('/', auth, async (req, res) => {
+  try {
+    const user = await User.findAll({
+      where: {
+        id: req.user.id
+      },
+      attributes: {
+        exclude: ['password']
+      }
+    });
+
+    const userData = user[0].dataValues;
+
+    _.forEach(userData, (value, key) => {
+      if (key !== _.camelCase(key)) {
+        userData[_.camelCase(key)] = value;
+        delete userData[key];
+      }
+    });
+
+    res.json(userData);
+  } catch (err) {
+    console.error(err.message);
+    res.status(500).send('Server Error');
+  }
+});
+
+router.post(
+  '/',
+  [
+    check('email', 'Please include a valid email').isEmail(),
+    check('password', 'Password is required').exists()
+  ],
+  async (req, res) => {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+      return res.status(400).json({ errors: errors.array() });
+    }
+
+    const { email, password } = req.body;
+
+    try {
+      let user = await User.findOne({ where: { email } });
+
+      if (!user) {
+        return res.status(400).json({ msg: 'Invalid Credentials' });
+      }
+
+      const checkPassword = await bcrypt.compare(password, user.password);
+
+      if (!checkPassword) {
+        return res.status(400).json({ msg: 'Invalid Credentials' });
+      }
+
+      if (!user.confirmed) {
+        return res.status(400).json({ msg: 'Email Confirmation Error' });
+      }
+
+      const payload = {
+        user: {
+          id: user.id
+        }
+      };
+
+      jwt.sign(
+        payload,
+        config.get('jwtSecret'),
+        {
+          expiresIn: 3600
+        },
+        (err, token) => {
+          if (err) throw err;
```

Comment:
Minor: stylistic thing, but at least in javascript, making sure there are proper block scoping is very useful in terms of making sure that the code is less bug prone. Otherwise, it gets extremely easier to have your code start to bleed through some cracks without realizing. This applies much more so for `switch` statements though.

```
if (err) {
    throw err;
}
```